modules/images/webp-uploads/load.php

```diff
@@ -105,9 +105,11 @@ function webp_uploads_create_sources_property( array $metadata, $attachment_id )
 		return $metadata;
 	}
 
+	$sizes_with_mime_type_support = webp_uploads_get_image_sizes_with_additional_mime_type_support();
+
 	foreach ( $metadata['sizes'] as $size_name => $properties ) {
-		// This image size is not defined or not an array.
-		if ( ! is_array( $properties ) ) {
+		// Do nothing if this image size is not an array or is not allowed to have additional mime types.
+		if ( ! is_array( $properties ) || ! isset( $sizes_with_mime_type_support[ $size_name ] ) ) {
```

Comment:
See my other comment. Depending what we decide there, this may need to be changed to
```suggestion
		if ( ! is_array( $properties ) || empty( $sizes_with_mime_type_support[ $size_name ] ) ) {
```

Reply:
Yep, updated.